pkgs/tools/misc/contacts/default.nix

```diff
@@ -9,15 +9,15 @@ stdenv.mkDerivation rec {
     sha256 = "0wdqc1ndgrdhqapvvgx5xihc750szv08lp91x4l6n0gh59cpxpg3";
   };
 
-  preBuild = ''
-    substituteInPlace Makefile --replace "xcodebuild" "/usr/bin/xcodebuild"
-  '';
+  buildInputs = [ xcbuild Foundation AddressBook ];
 
   installPhase = ''
     mkdir -p $out/bin
-    cp ./build/Deployment/contacts $out/bin
+    cp ./contacts-*/Build/Products/Default-*/contacts $out/bin
   '';
 
+  NIX_LDFLAGS = " -F${Foundation}/Library/Frameworks/ -F${AddressBook}/Library/Frameworks/";
```

Comment:
Surprised this is needed, since frameworks get a setup hook that does almost exactly this. It might pay to look into why that isn't happening, or maybe leave a comment saying "FIXME: wtf why isn't the framework setup hook working" 😄